### ISSUE TITLE
MacOS: Buggy Arrangement & Hover Functionality 

### ISSUE THREAD

**Issue by onexbash** [Shina-SG/Shina-Fox#20]
-- GREAT WORK! If that's your first time with FirefoxCSS, I'm impressed. --
 
I didn't dive into your code yet, but I guess the problem, that the arrangement on macOS is very weird, is because apple had the brilliant idea to not use 16:9 screens instead the ratio is about 15:10 (3024x1964), but that's also different between different macbook models lmao. 
Anyways, I guess you can get this fixed very easy by not using px or absolute positions in your css, to give it more flexibility.
The hover-only functionality is also a littlebit annoying for me, but maybe it just feels buggy on mac, I didn't try it on my linux machine yet.
If I find the time, I will try to fix it and send you a pull request, but I can't promise. 

Ah and additionally, it should be pretty easy to get it all packed in a curl command, so nobody has to configure anything beside's picking a theme. (I know it's easy anyways, but if you want to make it potentially accessible for everyone, that would be something you could think about.)  

![macOS_UI_Bug](https://github.com/Shina-SG/Shina-Fox/assets/118682177/26cbc927-1744-4fd1-bc3c-fbf62bcceea2)

**Comment by Shina-SG** [comment]
Funny enough I just watched a fireship video about css of why px is a bad practice and use em instead, let's me try doing em from now on. And for curl, I'm looking at other project of how they are doing to make this whole thing easier, from applying, updating, customizing, reverting back to an old ver, ect, curl could be a solution to all of that but I'm looking if an extension have the right api for the job since that would be really nice

**Comment by onexbash** [comment]
exactly. That's the best practice, cause em is relative to it's parent.
Things like text-center etc. that are also relative are good to use as well and sometimes I even use percentages of the screen-width. But I don't know what's possible in FirefoxCSS vs actual CSS code. Can you use everything that CSS has to give without exceptions?

**Comment by Shina-SG** [comment]
the exception probably is you need understand each of the element to not break it, since we can't create new stuff we can only inject stuff into existing stuff, for now I think css is enough for a few more version of my theme, but in the future we might need an extension on top of the theme as well.
Oh yeah and btw I don't support sidebery on the right yet too forgot to mention that only on the left for now, should be out on the next version since I didn't expect this much of attention early

**Comment by onexbash** [comment]
Yea it's hard, because you can't use frameworks like tailwind, you need to right plain CSS, that's tough. 
And Firefox is not even happy about people doing such things, that's why they call it "deprecated" and try things like Firefox Color, but that's not even close to what we want 🤣
